android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TiTableViewRowProxyItem.java

```diff
@@ -43,7 +43,7 @@
 	// every time we add a row. No sense checking it each time.
 	private static boolean ICS_OR_GREATER = (Build.VERSION.SDK_INT >= TiC.API_LEVEL_ICE_CREAM_SANDWICH);
 
-	private static final int LEFT_MARGIN = 5;
+	private static final int LEFT_MARGIN = 22;
```

Comment:
finally solved it :) Stupid dp vs pixels. Now both images (listview/tableview) have a 6dp padding